src/pricing.njk

```diff
@@ -4,46 +4,114 @@ align: center
 layout: layouts/page.njk
 sitemapPriority: 0.90
 meta:
-    description: Find out how much it costs to use FlowForge.
+description: Find out how much it costs to use FlowForge.
 ---
 
-<div class="px-6 pricing-tiles flex flex-col md:grid mb-16 md:max-w-screen-lg">
-    <div class="pricing-tile pt-4 pb-6 px-8">
-        <div class="flex flex-col justify-between items-center mb-4 sm:px-4 md:flex-row md:-mb-0 md:px-0 md:block">
-            <h4>Open-Source</h4>
-            <div class="mt-4">
-                <h2>$0</h2>
-                <label class="sub-label">Apache 2.0 Licensed<label>
-            </div>
-            <p class="mt-4">We are an open-core company, use FlowForge for free, always, with deployment options including Docker and Kubernetes.</p>
-            <a class="ff-btn ff-btn--secondary uppercase" href="/docs/install" onclick="capture('cta-install', {'position': 'primary'})">Install now</a>
-        </div>
+<div class="px-6 pricing-tiles flex flex-col mb-16 md:max-w-screen-lg">
+    <div class="flex px-6 -mb-6 -mt-32 mx-auto text-gray-400 md:max-w-screen-lg md:-mt-40 md:mb-1 md:block">
+        <p class="lead-p max-w-screen-lg m-auto text-center md:max-w-xl md:px-4">
+            Plans for every project. Whether you're an individual contributor, an IIoT manager, or an IT department, you'll find a suitable option to take your Node-RED applications to the next level.
```

Comment:
I think individual contributor is a phrase only managers use, not engineers themself?

```suggestion
            Plans for every project. Whether you're an engineer, an IIoT manager, or an IT department, you'll find a suitable option to take your Node-RED applications to the next level.
```

Reply:
I was thinking more broadly, not exclusively engineers but hobbyist too...
Taking that into consideration, what do you think? Should we add "engineer" or replace the "individual contributor".

Reply:
But than it's makers or practitioners?